rust/gui-client/src-tauri/tauri.conf.json

```diff
@@ -38,7 +55,7 @@
   "mainBinaryName": "firezone-client-gui",
   "identifier": "dev.firezone.client",
   "plugins": {},
-  "productName": "firezone-client-gui",
+  "productName": "Firezone",
```

Comment:
The application will now just show up as "Firezone" in the app drawer / menu on Linux. This is consistent with what we call it on all other platforms.